cabby/data/extract_wikigeo_contexts.py

```diff
@@ -28,53 +32,46 @@
 from cabby.data import extract
 from cabby.geo import regions
 
-
-FLAGS = flags.FLAGS
 flags.DEFINE_enum(
   "region", None, regions.ALLOWED_REGIONS,
-  "Map areas: Manhattan, Pittsburgh or Pittsburgh_small.")
-flags.DEFINE_string("path", None, "The path where the data will be saved.")
-
+  "Map areas: Manhattan, Pittsburgh or DC.")
```

Comment:
I removed DC query as the current DC region doesn't return Wikidata items. 
Instead - Pittsburgh_small

Reply:
Changed example to Pittsburgh_small

Reply:
I still see - "Map areas: Manhattan, Pittsburgh or DC."